index.js

```diff
@@ -0,0 +1,46 @@
+function calculate(arr) {
+  const operands = []
+  let operation = null
+  const findOperation = ['+', '-', '*', '/']
+
+
+  for (let i = 0; i < arr.length; i++) {
+    if (Number.isInteger(arr[i]) || Number.isInteger(parseInt(arr[i]))) {
+      operands.push(arr[i])
+    }
+    else if (arr[i] === null) {
+      operands.push(0)
+    }
+    else if (isOperation(findOperation, arr[i])) {
+      operation = arr[i]
+    }
+  }
+
+  if (findOperation.indexOf(operation) === -1) {
+    return NaN
+  }
+
+  switch (operation) {
+    case '*':
+      return operands[0] * operands[1]
+
+    case '/':
+      return operands[0] / operands[1]
+
+    case '+':
+      return operands[0] + operands[1]
+
+    case '-':
+      return operands[0] - operands[1]
```

Comment:
You should be typecasting into a number here or above when building the array. `'2'` would pass the first check above but still be inserted as `'2'` and `'2' + 2` is `'22'`